community/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/compatibility/v3_3/runtime/pipes/SortPipe.scala

```diff
@@ -27,30 +29,21 @@ case class SortPipe(source: Pipe, orderBy: Seq[SortDescription])
                    (val id: Id = new Id)
   extends PipeWithSource(source) {
   protected def internalCreateResults(input: Iterator[ExecutionContext], state: QueryState): Iterator[ExecutionContext] = {
+    assert(orderBy.nonEmpty)
```

Comment:
since `orderBy` is an instance field, can't we do this assertion outside the method?